docs/source/getting_started/index.rst

```diff
@@ -4,4 +4,49 @@
 Introduction
 ============
 
-TODO: Do this section.
\ No newline at end of file
+This article will help you figure out how to start using Ink with GDScript in Godot.
+
+What is Inkgd?
+==============
+
+Inkgd is Ink for Godot, more precisely GDScript. It's a GDScript API that you import as an addon in your Godot project that can read an Ink story (file with .ink.json suffix) and execute Ink's methods directly in GDScript. With Inkgd you'll be able to load your story and get the text from each knot, choice and diverts, then use the text as-is. It also binds variable changes and external functions with custom functions found in your Godot scripts.
+
+A normal use of Inkgd would be to load a compiled story in '.ink.json' and then call "continue_story" until a branch is found. Here the choices are read as an array of strings and the story waits for you to input a choice. Once you submit the selected choice (represented by the index of the choices array) to the story, you repeat the process (i.e. keep calling "continue_story" until a branch is found).
+
+Remember that this API only provides the text, it does not provide a graphical representation of the Ink story. That's up to you.
+
+Installing the addon
+====================
+
+You have two ways of installing this addon: the easy and convenient way or the slightly more difficult way.
+
+The easy way
+************
+
+Open a new Godot project, then navigate to "AssetLib" on the top of the screen, search for "inkgd". Select "inkgd" from author "ephread", in the popup screen click "Download". Once the addond is downloaded to your computer a popup screen will show up, this describes the new files it'll add to your project. By default it should select everything, if this is not the case then select all files, click "Install". After the installation is done a popup message will show saying the addon was added to your project, click "Ok" to close the popup.
```

Comment:
Once the _addond_ […] → Once the _addon_ […]

Reply:
Done.